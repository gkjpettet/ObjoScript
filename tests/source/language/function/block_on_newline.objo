function test() 
{ // expect error.
System.print("ok")
}