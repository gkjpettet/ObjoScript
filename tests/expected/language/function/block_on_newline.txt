Expected a `{` after function parameters.